lib/uo_cli.rb

```diff
@@ -451,7 +458,7 @@ def self.create_project_folder(dir_name, empty_folder = false, overwrite_project
         else
           abort("\nNo OpenDSS results available in folder '#{opendss_folder}'\n")
         end
-      elsif @opthash.subopts.to_s.include?('reopt')
```

Comment:
@vtnate I was running into an error because this was returning true always, I have modified this a bit to resolve that

Reply:
👍